src/pages/vrf/v2/subscription/examples/test-locally.mdx

```diff
@@ -0,0 +1,160 @@
+
+1. Under _DEPLOY_, fill in `SUBSCRIPTIONID` with your subscription id, `vrfCoordinator` with the deployed _VRFCoordinatorV2Mock_ address and, _KEYHASH_ with an arbitrary `bytes32` (In this example, you can set the _KEYHASH_ to `0xd89b2bf150e3b9e13446986e571fb9cab24b13cea0a43ea20a6049a85cc807cc`).
+
+1. Click on _transact_ to deploy the _RandomNumberConsumerV2_ contract.
+
+1. Once deployed, you should see the _RandomNumberConsumerV2_ contract under _Deployed Contacts_.
+
+   <ClickToZoom src="/images/vrf/mock/v2-deployedconsumer.jpg" />
+
+1. Note the address of the deployed contract.
+
+### Add the consumer contract to your subscription
+
+1. Under _Deployed Contracts_, open the functions list of your deployed _VRFCoordinatorV2Mock_ contract.
+
+1. Click on _addConsumer_ and fill in the `_subid` with your subscription id and `_consumer` with your deployed consumer contract address.
+
+   <ClickToZoom src="/images/vrf/mock/v2-addconsumer.jpg" />
+
+1. Click on _transact_.
+
+### Request random words
+
+1. Under _Deployed Contracts_, open the functions list of your deployed _RandomNumberConsumerV2_ contract.
+
+1. Click on `requestRandomWords`.
+
+   <ClickToZoom src="/images/vrf/mock/v2-requestrandomwords.jpg" />
+
+1. In the RemixIDE console, read your transaction logs to find the VRF request id. In this example, the request id is _1_.
+
+   <ClickToZoom src="/images/vrf/mock/v2-requestrandomwordslogs.jpg" />
+
+1. Note your request id.
+
+### Fulfill the VRF request
+
+Because you are testing on a local blockchain environment, you must fulfill the VRF request yourself.
+
+1. Under _Deployed Contracts_, open the functions list of your deployed _VRFCoordinatorV2Mock_ contract.
+
+1. Click `fulfillRandomWords` and fill in `_requestId` with your VRF request id and `_consumer` with your consumer contract address.
+
+   <ClickToZoom src="/images/vrf/mock/v2-fulfillrandomwords.jpg" />
+
+1. Click on `transact`.
+
+### Check the results
+
+1. Under _Deployed Contracts_, open the functions list of your deployed _RandomNumberConsumerV2_ contract.
+
+1. Click on `s_requestId` to display the last request id. In this example, the output is _1_.
+
+   <ClickToZoom src="/images/vrf/mock/v2-deployedconsumerrequestid.jpg" />
+
+1. Each time you make a VRF request, your consumer contract requests two random words. Once the request is fulfilled, the two random words are stored in the `s_randomWords` array. You can check the stored random words by reading the two first indexes of the `s_randomWords` array. To do so, click on the _s_randomWords_ function and:
+
+   1. Fill in the index with _0_ then click on _call_ to read the first random word.
+
+      {" "}
+
+      <ClickToZoom src="/images/vrf/mock/v2-firstrandomword.jpg" />
+
+   1. Fill in the index with _1_ then click on _call_ to read the second random word.
+
+      {" "}
+
+      <ClickToZoom src="/images/vrf/mock/v2-secondrandomword.jpg" />
+
+## Conclusion
```

Comment:
Nit: I haven't seen most docs tutorials end with a section actually titled "Conclusion" - my suggestion is to retitle this "Next steps" if it doesn't fit well into the usual structure for our "What's next" sections.

After making that change, we would avoid adding a "What's next" section to these specific pages to avoid duplication.